ts/Extensions/Annotations/Popup.ts

```diff
@@ -218,18 +218,22 @@ H.Popup.prototype = {
         let _self = this,
             closeBtn: HTMLDOMElement;
 
+        const iconsURL = this.iconsURL;
+
         // create close popup btn
         closeBtn = createElement(DIV, {
             className: PREFIX + 'popup-close'
         }, null as any, this.container);
 
         closeBtn.style['background-image' as any] = 'url(' +
-                this.iconsURL + 'close.svg)';
+                (
+                    iconsURL.match(/png|svg|jpeg|jpg|gif/ig) ?
+                        iconsURL : iconsURL + 'close.svg'
+                ) + ')';
 
         ['click', 'touchstart'].forEach(function (eventName: string): void {
             addEvent(closeBtn, eventName, function (): void {
-
-                fireEvent(_self.chart.navigationBindings, 'closePopup');
+                _self.closePopup();
```

Comment:
We need a `fireEvent` here. @pawellysy could you remind us what was the reason?

Reply:
Can we leave both calls?

Reply:
This line was added because clicking on the 'x' button only hid the popup, and didn't call the event. 
here is the issue: https://github.com/highcharts/highcharts/issues/14280

Reply:
So can we leave both? ClosePopup and fireEvent? It will be a little bit more flexible for other popups.

Reply:
I'm not sure, let's try it

Reply:
I think that it can be both 👍

Reply:
Done